README.md

```diff
@@ -49,6 +49,89 @@ TypeScript typings. Types are located under the `models` subfolder:
 import type TYPE_NAME from "@jellyfin/client-axios/models";
 ```
 
+## Usage
+
+Install it in your project:
+
+```bash
+npm i @jellyfin/client-axios
+```
+
+This library follows the API pattern you can get on https://api.jellyfin.org/ or on your own server at `/api-docs/swagger/index.html` or `/api-docs/redoc/index.html`.
+
+### Basic example
+
+```typescript
+// Get server infos
+import { SystemApi } from "@jellyfin/client-axios";
+
+const systemApi = new SystemApi(
+  undefined, // OpenAPI configuration object, found in `configuration.ts`
+  "https://demo.jellyfin.org/stable", // Base URL
+  undefined // Axios base object to use for queries, can be used to set base URL or headers
+);
+const infos = (await systemApi.getPublicSystemInfo()).data;
+console.log(infos);
+```
+
+### Authentication
+
+```typescript
+import { v4 } from "uuid"; // Used to generate random string
+import { UserApi } from "@jellyfin/client-axios";
+
+const base_token = `MediaBrowser Client="My New Client", Device="${
+  window.navigator.userAgent
+}", DeviceId="${v4()}", Version="1.0.0"`;
+
+const vanilla_token = `${base_token}, Token=""`; // We need a vanilla token to authenticate
+
+const ax = axios.create({
+  headers: {
+    "X-Emby-Authorization": vanilla_token
```

Comment:
I'd suggest using the `Authorization` header as that's what all clients are slowly migrating to because it's a HTTP standard unlike `X-Emby-Authorization`. The value can be kept the same ("MediaBrowser" as the scheme with the parameters after it)

Reply:
It seems like the openapi spec uses `X-Emby-Authorization` still. I planned to keep using that until the spec was updated. :man_shrugging:

Reply:
I just made a PR for that; https://github.com/jellyfin/jellyfin/pull/6556